### DISCUSSION TITLE
PC Usb Support

### DISCUSSION THREAD

**Discussion by psicotron3** [darthcloud/BlueRetro#598] [Ideas]
I've been thinking if making a cable to use BlueRetro on PC as a wired XInput controller would be beyond the scope of the project.

At first glance, using blueretro as a "bridge" to connect a controller to pc that already can be just plugged in directly via usb or paired by Bluetooth looks useless.

The controller support for PC is really good only if you're using a XInput controller. 

We can connect Ds4, Ds5 and Switch Pro controller directly to PC but not all games will work out of the box and we need to use other software like BetterJoy, mess with usb drivers and a lot of issues.

Using blueretro with PC via a USB cable would fix this issue. 

Since there are commercial options that already  do exactly this, I don't know if the work needed to adapt blueretro would be worth it (or maybe the esp32 simply can't do that).

**Comment by hdz-dev** [comment]
Within the retro world there are also emulation enthusiasts, it would be great if blueretro allowed original controls modified for bluetoot to be connected via USB to the PC, also having compatibility with consoles would be the ultimate adapter

**Comment by MexChiKilin** [comment]
can the esp32 work on a pc as a multi-Bluetooth devices as any Bluetooth dongle to play retrogame emulators?
